crates/starknet-devnet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -466,10 +466,14 @@ impl JsonRpcHandler {
     /// devnet_getPredeployedAccounts
     pub async fn get_predeployed_accounts(
         &self,
-        params: PredeployedAccountsQuery,
+        params: Option<PredeployedAccountsQuery>,
     ) -> StrictRpcResult {
-        let predeployed_accounts =
-            get_predeployed_accounts_impl(&self.api, params).await.map_err(ApiError::from)?;
+        let predeployed_accounts = get_predeployed_accounts_impl(
+            &self.api,
+            params.map_or(PredeployedAccountsQuery { with_balance: Option::None }, |val| val),
```

Comment:
Why not use `unwrap_or` instead of `map_or`?

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/7b59ae65789b9555bc2c1597a4ce6d4821bb8f54